java/src/jmri/jmrit/signalling/AddEntryExitPairPanel.java

```diff
@@ -135,10 +135,14 @@ private void addButton() {
 
     private void autoDiscovery() {
         if (!InstanceManager.getDefault(jmri.jmrit.display.layoutEditor.LayoutBlockManager.class).isAdvancedRoutingEnabled()) {
-            int response = JOptionPane.showConfirmDialog(null, Bundle.getMessage("EnableLayoutBlockRouting"));
+            int response = JOptionPane.showConfirmDialog(null,
```

Comment:
If I were doing a major change in this package I might merge the 3 resource bundles and change all of the references to getMessage().

In this case, I am fixing a Java exception.  My normal process is to create signal mast logic before creating the NX setup.  This enables block routing so the NX code is never executed.  It is a random chance that I discovered this problem by testing NX before creating signals and signal mast logic.